packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceSourceRetrieveManifest.test.ts

```diff
@@ -11,8 +11,7 @@ import { expect } from 'chai';
 import * as path from 'path';
 import { createSandbox, SinonStub } from 'sinon';
 import { LibrarySourceRetrieveManifestExecutor } from '../../../src/commands/forceSourceRetrieveManifest';
-import { workspaceContext } from '../../../src/context';
-import { nls } from '../../../src/messages';
```

Comment:
@jeffb-sfdc was NLS meant to be removed here too?

Reply:
nls isn't used, so I removed it since workspaceContext (also not used) was removed.